pages/common/lwp-request.md

```diff
@@ -0,0 +1,27 @@
+# lwp-request
+
+> Simple Perl-based command-line HTTP client.
```

Comment:
@waldyrious 
it is [libwww-perl](http://search.cpan.org/~ether/libwww-perl-6.15/lib/LWP.pm). I am not sure if this is useful information.